packages/jellyfish-api-core/src/category/icxorderbook.ts

```diff
@@ -0,0 +1,146 @@
+import { BigNumber, ApiClient } from '../.'
+
+/**
+ * icxorderbook RPCs for DeFi Blockchain
+ */
+export class ICXOrderBook {
+  private readonly client: ApiClient
+
+  constructor (client: ApiClient) {
+    this.client = client
+  }
+
+  /**
+   * Create and submits an ICX order creation transaction.
+   *
+   * @param {ICXOrder} order
+   * @param {string} [order.tokenFrom] Symbol or id of selling token
+   * @param {string} [order.chainFrom] Symbol or id of selling chain
+   * @param {string} [order.chainTo] Symbol or id of buying chain
+   * @param {string} [order.tokenTo] Symbol or id of buying token
+   * @param {string} [order.ownerAddress] Address of DFI token for fees and selling tokens in case of DFC/BTC order type
+   * @param {string} [order.receivePubkey] pubkey which can claim external HTLC in case of EXT/DFC order type
+   * @param {BigNumber} [order.amountFrom] tokenFrom coins amount
+   * @param {BigNumber} [order.orderPrice] Price per unit
+   * @param {number} [order.expiry=2880] Number of blocks until the order expires, default 2880 DFI blocks
+   * @param {UTXO[]} inputUTXOs Specific utxos to spend
+   * @param {string} [inputUTXOs.txid] transaction Id
+   * @param {number} [inputUTXOs.vout] The output number
+   * @return {Promise<ICXGenericResult>} Object including transaction id of the the result transaction
+   */
+  async createOrder (order: ICXOrder, inputUTXOs: UTXO[] = []): Promise<ICXGenericResult> {
```

Comment:
```suggestion
   * @param {UTXO[]} utxos Specific utxos to spend
   * @param {string} [utxos.txid] transaction Id
   * @param {number} [utxos.vout] The output number
   * @return {Promise<ICXGenericResult>} Object including transaction id of the the result transaction
   */
  async createOrder (order: ICXOrder, utxos: UTXO[] = []): Promise<ICXGenericResult> {
```

Reply:
I think after resolving this, we are good to go